common/src/main/java/org/apache/falcon/metadata/InstanceRelationshipGraphBuilder.java

```diff
@@ -280,20 +271,15 @@ public void addImportedInstance(WorkflowExecutionContext context) throws FalconE
                 feedInstanceDataPath, sourceClusterName, datasourceName);
         String feedInstanceName = getFeedInstanceName(feedName, sourceClusterName,
                 feedInstanceDataPath, context.getNominalTimeAsISO8601());
-        Vertex feedInstanceVertex = findVertex(feedInstanceName, RelationshipType.FEED_INSTANCE);
-
-        LOG.info("Vertex exists? name={}, type={}, v={}",
-                feedInstanceName, RelationshipType.FEED_INSTANCE, feedInstanceVertex);
-        if (feedInstanceVertex == null) { // No record of instances NOT generated by Falcon
-            LOG.info("{} instance vertex {} does not exist, add it",
-                    RelationshipType.FEED_INSTANCE, feedInstanceName);
-            feedInstanceVertex = addFeedInstance(// add a new instance
-                    feedInstanceName, context, feedName, context.getSrcClusterName());
-        }
+        Vertex feedInstanceVertex = addFeedInstance(
+                feedInstanceName, context, feedName, context.getSrcClusterName());
+
+        Map<RelationshipProperty, String> properties = new HashMap<RelationshipProperty, String>();
+        properties.put(RelationshipProperty.TIMESTAMP, context.getTimeStampAsISO8601());
```

Comment:
The edge is populated with nominal time alone and not the status. Let us add status to the edge to be consistent with other entity operations so that the imported instances are searchable.